src/input/mod.rs

```diff
@@ -1153,6 +1169,10 @@ impl State {
         pattern: KeyPattern,
         direction: Option<Direction>,
     ) {
+        if self.common.session_lock.is_some() {
+            return;
+        }
```

Comment:
We probably should at least allow `Action::Terminate`. There is also a point to be made to allow `Action::Debug`.

Reply:
`Debug` makes sense if normal releases won't be built with the `debug` feature anyway. Terminating the compositor is a sensitive operation if the compositor was started manually from a tty, in which case it will drop you to a logged in console. If that's a concern. Otherwise it should go back to the login screen, but will kill the user's processes.

Reply:
> Terminating the compositor is a sensitive operation if the compositor was started manually from a tty

Right, can we somehow determine this or at least allow this, when running cosmic-greeter as a display manager? I feel like you want to be able to kill a misbehaving session.